docker/add_gitbase.py

```diff
@@ -17,5 +18,32 @@ def get_or_create_gitbase_db():
     db.session.add(dbobj)
```

Comment:
Shouldn't this be added and committed only `if not dbobj`?

Reply:
no. Because it updates `GITBASE_DATABASE_URI`.
I might do extra update queries but we run it only once on init so it's not a problem.
Also it works the same as superset's data example scripts.